python/dgl/transform.py

```diff
@@ -383,6 +384,137 @@ def reverse(g, share_ndata=False, share_edata=False):
         g_reversed._edge_frame = g._edge_frame
     return g_reversed
 
+def reverse_heterograph(g, share_ndata=True, share_edata=False):
+    r"""Return the reverse of a graph
+
+    The reverse (also called converse, transpose) of a graph with edges
+    :math:`(i_1, j_1), (i_2, j_2), \cdots` is a new graph with edges
+    :math:`(j_1, i_1), (j_2, i_2), \cdots`.
+
+    For a heterograph with multiple edge types, we can treat edges corresponding to each
+    type as a separate graph and compute the reverse for each of them.
+
+    Given a :class:`DGLGraph` object, we return another :class:`DGLGraph` object
```

Comment:
For cross-page reference, I think we need to replace `:class:`DGLGraph`` by `:class:`dgl.DGLGraph``